lib/capistrano/data_migrate/migrate.rb

```diff
@@ -7,10 +7,9 @@
       conditionally_migrate = fetch(:conditionally_migrate)
       info '[deploy:migrate] Checking changes in db/migrate or db/data' if conditionally_migrate
 
-      if conditionally_migrate && (
-          test("diff -q #{release_path}/db/migrate #{current_path}/db/migrate") ||
-          test("diff -q #{release_path}/db/data #{current_path}/db/data")
-        )
+      if conditionally_migrate &&
+         test("diff -q #{release_path}/db/migrate #{current_path}/db/migrate") &&
+         test("diff -q #{release_path}/db/data #{current_path}/db/data")
```

Comment:
Are you okay with this change? @ilyakatz

Reply:
yes, the change looks reasonable to me. @EnomaDebby @borama can you add release notes please? this may need to be a major upgrade for the versions since it's not backward compatible. (small but i think it fits)

Reply:
also, do you know why the build is failing?

Reply:
@ilyakatz @EnomaDebby Regarding the failing builds, it seems to me that it's the same issue as is resolved in [this commit](https://github.com/rails/activerecord-session_store/commit/46ea4748d280799781f2c25fc981ad0e234ec297) in the ActiveRecordSessionStore gem, see also [their PR](https://github.com/rails/activerecord-session_store/pull/141). I.e. you probably need to lock sqlite3 gem version to 1.3.6 in the CI build gemfiles.

I came to this conclusion after seeing [this line](https://travis-ci.org/ilyakatz/data-migrate/jobs/492846760#L908) in the CI build. I can try to make another PR with a fix if you like.

Reply:
got it, @EnomaDebby want to help getting this through or want me to?